src/pages/SetDefaultWorkspacePage.tsx

```diff
@@ -37,12 +37,12 @@ function SetDefaultWorkspacePage({route}: SetDefaultWorkspacePageProps) {
     const session = useSession();
 
     const selectPolicy = (selectedPolicyID?: string) => {
-        if (!selectedPolicyID) {
+        if (!selectedPolicyID || !navigateTo) {
             return;
         }
         // eslint-disable-next-line rulesdir/no-default-id-values
         setNameValuePair(ONYXKEYS.NVP_ACTIVE_POLICY_ID, selectedPolicyID, activePolicyID ?? '');
-        Navigation.goBack(backTo);
+        Navigation.navigate(navigateTo);
```

Comment:
Why are we using navigate ?

Reply:
For example, in a case where user doesn't have a default workspace we cannot navigate him back to `category` page as he wasn't there yet.
It looks something like this CreateExpense -> SetDefaultWorkpsace -> CreateExpense/Category

Reply:
Where if we were to use a goBack it would navigate user back to CreateExpense

CreateExpense -> SetDefaultWorkpsace -> (backTo) CreateExpense